src/main/java/com/epam/reportportal/api/RequestBuilder.java

```diff
@@ -1,6 +1,106 @@
 package com.epam.reportportal.api;
 
+import io.restassured.http.ContentType;
+import io.restassured.response.Response;
+import io.restassured.specification.RequestSpecification;
+
+import static com.epam.reportportal.utils.configuration.EnvironmentConfiguration.barearToken;
+import static io.restassured.RestAssured.given;
+import static org.springframework.http.HttpStatus.*;
+
 public class RequestBuilder {
 
+    public static RequestSpecification requestSpecificationWithAuth() {
+        return given()
+                .accept("*/*")
+                .header("Authorization", barearToken());
+    }
+
+    public static RequestSpecification requestSpecificationWithAuthContentTypeJsonAndBody(String body) {
+        RequestSpecification requestSpecification = requestSpecificationWithAuth()
+                .contentType(ContentType.JSON)
+                .body(body);
+        return requestSpecification;
+    }
+
+    public static RequestSpecification incorrectRequestSpecification() {
+        String incorrectToken = "Bearer yJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE3MTM0NjY2OTksInVzZXJfbmFtZSI6ImRhcmlhIiwiYXV0a";
+        return given()
+                .accept("*/*")
+                .header("Authorization", incorrectToken);
+    }
+
+    public static <T> T getWithOkStatusCodeAndResponse(String url, Class<T> clazz) {
```

Comment:
it breaks the single responsibility principle.
does the status code really metter here?